src/normalizer.ts

```diff
@@ -154,7 +154,8 @@ export type InternalMasterSlide = {
 export type InternalPresentation = {
   slides: InternalSlide[];
   masterSlides: { [name: string]: InternalMasterSlide };
-  layout: "16x9" | "16x10" | "4x3" | "wide";
+  layout: "16x9" | "16x10" | "4x3" | "wide" | "custom";
```

Comment:
The slightly annoying property of this is that you can have `<Presentation layout="16x9" customLayout={{width: 100, height: 10}}>` without any complaints. What if there was just one property and the type was something like `"16x9" | "16x10" | "4x3" | "wide" | { width: number; height: number }`?

Reply:
@wyozi Nice suggestion!